domino.go

```diff
@@ -283,6 +317,17 @@ type KeyValue struct {
 /************************************** GetItem ****************************************/
 /***************************************************************************************/
 type get dynamodb.GetItemInput
+type output struct {
+	*dynamodb.GetItemOutput
+	Error error
+}
+
+func (o *output) Result(item interface{}) (err error) {
```

Comment:
This is the key idea. It borrows from the go-redis library by returning an object to wrap underlying data, while providing access to both the dynamo output as well as facilities for deserializing the data.